### ISSUE TITLE
Allow user to tab into date/time inputs

### ISSUE THREAD

**Issue by bookis** [dggpoliticalaction/in-house-mgmt#193]
> Very minor accessibility/UX on this form, tabbing between inputs does not work correctly. Date fields should be valid focus inputs.
> 
> <img width="803" height="630" alt="Image" src="https://github.com/user-attachments/assets/d0225e7a-34ae-44f6-880e-abed01120ef0" />
> 
> 👍 

 _Originally posted by @IHaveEatenFoliage in [#177](https://github.com/digitalgroundgame/in-house-mgmt/issues/177#issuecomment-4252014397)_
